triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -139,10 +139,10 @@ abstract class AbstractFuncTests
           result <- finalStateF.map(toResult)
           acs <- queryACS(client, party)
         } yield {
-          assert(result.activeAssets == Seq().toSet)
-          assert(result.successfulCompletions == 4)
-          assert(result.failedCompletions == 0)
-          assert(acs(assetMirrorId).size == 2)
+          result.activeAssets shouldBe empty
+          result.successfulCompletions should ===(4)
```

Comment:
Can we just use `shouldBe` for both of these and everywhere else in place of `should===`?

Reply:
If we were to standardize, I would prefer to standardize on `should ===` rather than `shouldBe`, as the latter offers no way to enforce type constraints. (The default of `===` doesn't, but that is a different question.)

Reply:
If you're not talking about standardizing, though, I can make this one either way.